freqtrade/rpc/api_server/api_schemas.py

```diff
@@ -95,6 +95,7 @@ class Profit(BaseModel):
     avg_duration: str
     best_pair: str
     best_rate: float
+    short_trades: int
```

Comment:
For the `TradeSchema`, `is_short` makes perfect sense ... here, i'm not sure what the intention is (how you intend to use it afterwards).

if you think of showing a "amount of short trades" - then i think we should also add a `long_trades` alongside, as `trade_count` is the total of trades (both long and short).
Without that, we'd have to always subtract one from the other, which doesn't really make sense.

Reply:
Sure, I think it might be too detailed. I will remove the short_trades for now. 

Probably another PR in the future when features are more complete to test. I will create another endpoint for more detailed report.